mesh/settings.py

```diff
@@ -12,7 +12,10 @@
 
 from pathlib import Path
 import os
+from dotenv import load_dotenv
 
+
+load_dotenv() 
```

Comment:
this should not be needed, remove this, try again, and if it still doesn't work contact me

Reply:
okok